proposals/Level-1.md

```diff
@@ -93,19 +92,19 @@ Each exception type has a `type signature`. The type signature defines the list
 of values associated with the exception.
 
 Within the module, exception types are identified by an index into the
-[exception index space](#exception-index-space). This index is referred to as
-the `exception tag`. The `tagged exception type` is the corresponding
+[exception index space](#exception-index-space). This static index refers to the
+corresponding runtime index that uniquely identifies the exception type, and is
```

Comment:
Suggestion: Maybe it's even clearer if you avoid calling the runtime thing an "index" in the first place. How about changing wording to use "tag" right away?

Reply:
Sound good. Changing.